tests/unit/sanakoyeu_et_al_2018/test_modules.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+
+import torch
+from torch import nn
+
+import pystiche_papers.sanakoyeu_et_al_2018 as paper
+from pystiche import misc
+from pystiche_papers.utils import ResidualBlock, same_size_padding
+
+
+def test_get_padding(subtests):
+    kernel_size = 3
+    for str_padding, desired in (
+        ("same", same_size_padding(kernel_size)),
+        ("valid", 0),
+    ):
+        with subtests.test(str_padding):
+            actual = paper.get_padding(str_padding, kernel_size)
+            assert actual == desired
+
+
+def test_get_padding_wrong_string():
+    kernel_size = 3
+    with pytest.raises(ValueError):
+        paper.get_padding("", kernel_size)
```

Comment:
You don't need to test this, since the behavior is handled in `pystiche.misc.verify_str_arg`. As a rule of thumb you only need to test `Exception`s that are raised by our code.